.github/workflows/cli-binary-pr.yaml

```diff
@@ -0,0 +1,45 @@
+name: Build wgc binaries
+on:
+  pull_request:
+    paths:
+      - "cli/**/*"
+      - "connect/**/*"
+      - ".github/workflows/cli-ci.yaml"
+
+permissions:
+  contents: write
+  packages: write
+
+jobs:
+  wgc-releases-matrix:
+    name: Build wgc binaries
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+
+    strategy:
+      matrix:
+        target: [bun-linux-x64, bun-linux-arm64, bun-darwin-x64, bun-darwin-arm64]
```

Comment:
Would windows work as well?

Reply:
Yes for windows x64, but the install script won't unless I write powershell (I do not want to write powershell)

Reply:
Or, it will build, I'm not sure if it will work

Reply:
Let's solve once thing at a time. I have Windows, I can test it.

Reply:
https://github.com/wundergraph/cosmo/actions/runs/13517613280?pr=1620 windows builds